### PR TITLE
Feature/detections

### PR DESCRIPTION
**Pull request by Crisfole**
Add feature detection POST

### CODE REVIEW

File: src/api/args.ts [hunk 1]
```diff
@@ -63,3 +63,13 @@ export type UsersQuery = Query & {
...
 };
+
+export type ObservePlateQuery = Query & {
+  scope?: string;
+  alpr?: 'openalpr' | 'sighthound' | 'platerecog';
+  method?: 'scanner';
+  filename: string;
+  latitude: number;
+  longitude: number;
+  accuracy: number;
```
Comment (japfohl) [inline src/api/args.ts:74]: I _assume_ we're using the geolocation API.  Any reason we're not just passing the [GeolocationCoordinates](https://developer.mozilla.org/en-US/docs/Web/API/GeolocationCoordinates)?

**Review by japfohl — APPROVED** [review]
Looks good.

File: src/api/args.ts (hunk 1 above)
Comment (Crisfole) [reply src/api/args.ts:74]: Cuz...

Comment (japfohl) [reply src/api/args.ts:74]: Best reason I've heard all day.

Comment (Crisfole) [reply src/api/args.ts:74]: I'm merging, but we can revisit this later.

**Pull request merged**